docs/docs.trychroma.com/pages/integrations/spacy.md

```diff
@@ -0,0 +1,18 @@
+---
+---
+
+# spaCy
+
+Pre-trained embeddings that are available from [spaCy](https://spacy.io/models/) can be used for encoding text into vectors. They are fast, robust and good alternative for a lot of language models. To use spacy models in embedding function we have to install spacy module and also download a model of our choice. Please use the below snippet to install and download a model of our choice.
+
+```pip install spacy```
```

Comment:
maybe add this in a ```bash``` block for better readability.

Reply:
Resolved changes, let me know if any resolutions is required.